modules/n1ql/pages/n1ql-language-reference/condfunnum.adoc

```diff
@@ -4,32 +4,38 @@
 
 {description}
 
-IFINF(expression1, expression2, \...)
+== IFINF(expression1, expression2, \...)
 
+=== Description
 Returns first non-MISSING, non-Inf number.
 Returns MISSING or NULL if a non-number input is encountered first.
 
-IFNAN(expression1, expression2, \...)
+== IFNAN(expression1, expression2, \...)
 
+=== Description
 Returns first non-MISSING, non-NaN number.
 Returns MISSING or NULL if a non-number input is encountered first.
 
-IFNANORINF(expression1, expression2, \...)
+== IFNANORINF(expression1, expression2, \...)
 
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage example per function